### ISSUE TITLE
asynchronous export: invalid read syntax "#"

### ISSUE THREAD

**Issue by maxecharel** [polymode/poly-org#38]
I first upgraded org-mode to 9.5.2 since a fix for a similar issue has been introduced in org >= 9.5.  The fix works unless poly-org is used; in that case the issue remains (see [this org mailing list thread](https://list.orgmode.org/aw_wwlJcCuUcOWTA5vHxOQaat4W5s33Auov2jBSfrpkgCoM2JuGIUN6WNDQMfBYQL2NQog7cewykBnCPQa2ZuRr0Nc0mwyf4A_yEyYvxxZM=@protonmail.com/T/#t)). 
Could you have a look at that? Thx in advance.

**Comment by maikol-solis** [comment]
I have the same issue and reported here (https://github.com/nobiot/org-transclusion/issues/137)

Until today, I haven't discovered what causes this. 

Thanks for your help.

**Comment by mcmillanbb** [comment]
This issue should be fixed now, org made a patch in February.
See https://lists.gnu.org/archive/html/emacs-orgmode/2025-02/msg00265.html

**Comment by mcmillanbb** [comment]
No, I don't think it is polymode specific. I believe the issue was that 
1. polymode stores a lot of information as text properties, 
2. when org does asynchronous export, it saves the org buffer as a temporary file that is read into a new background emacs process. This saves various variables, as well as text properties.
3. On reading in the new process, some of the text properties from polymode were not getting parsed correctly, crashing the export process.

I think the change on org side was just to remove text properties from the export, since export shouldn't depend on them anyways.
